src/DacpacTool/PackageDeployer.cs

```diff
@@ -213,12 +213,12 @@ public void SetProperty(string key, string value)
                     "CompareUsingTargetCollation" => bool.Parse(value),
                     "CreateNewDatabase" => bool.Parse(value),
                     "DatabaseLockTimeout" => int.Parse(value),
-                    "DatabaseSpecification" => ParseDatabaseSpecification(value),
+                    "DatabaseSpecification" => PropertyParser.ParseDatabaseSpecification(value),
```

Comment:
Can't we replace this method with the `PropertyParser` to avoid duplication of logic?

Reply:
Done